ADCS/adcs_components.py

```diff
@@ -0,0 +1,68 @@
+"""
+Holds classes for each type of measurements on the ADCS
+"""
+# pylint: disable=too-few-public-methods
+# pylint: disable=useless-parent-delegation
+
+
+class ThreeDimensionalMeasurements:
```

Comment:
After learning more about Python, I have found that it would be better to make this a dataclass. I am not saying to change it in this pull request, but I think one of us should refactor this in the future.